Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -243,6 +243,43 @@ lemma pow_lt_pow_succ (ha : 1 < a) : a ^ n < a ^ n.succ := by
 lemma pow_lt_pow_right₀ (ha : 1 < a) (hmn : m < n) : a ^ m < a ^ n := by
   induction' hmn with n _ ih; exacts [pow_lt_pow_succ ha, lt_trans ih (pow_lt_pow_succ ha)]
 
+lemma zpow_lt_zpow_of_lt_one₀ (ha₀ : 0 < a) (ha₁ : a < 1) {m n : ℤ} (hmn : m < n) : a ^ n < a ^ m :=
+  zpow_right_strictAnti₀ ha₀ ha₁ hmn
```

Comment:
This is a special case of `zpow_lt_zpow_right_of_lt_one₀` and should be deleted

Reply:
I'm confused. The conclusion of your lemma [zpow_lt_zpow_right_of_lt_one](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/GroupWithZero/Unbundled.html#zpow_lt_zpow_right_of_lt_one%E2%82%80)₀ is `a ^ n ≤ a ^ m`, and not 
`a ^ n < a ^ m `. 
(From the name I would have expected the conclusion to be `a ^ n < a ^ m `.)
I do want the version with strict inequality.

Reply:
Whoops, looks like a typo. It should say `<`. Feel free to fix it in this PR.

Reply:
I think your lemma [zpow_lt_zpow_right_of_lt_one₀](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/GroupWithZero/Unbundled.html#zpow_lt_zpow_right_of_lt_one%E2%82%80) is in fact lemma [zpow_le_zpow_right_of_le_one₀](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/GroupWithZero/Unbundled.html#zpow_le_zpow_right_of_le_one%E2%82%80).

I'll fix this, so that `zpow_lt_zpow_of_lt_one₀` is what it should be.